lib/views/schedule/schedule.dart

```diff
@@ -12,59 +14,48 @@ class Schedule extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Livestream Schedule'),
-        backgroundColor: ThcColors.darkBlue,
         actions: [
-          Visibility(
-            visible: userType.isAdmin,
-            child: IconButton(
-              icon: const Icon(Icons.add), // Choose your desired icon
-              onPressed: () {
-                // Add your onPressed callback function here
-                // This function will be called when the icon button is clicked
-                navigator.push(const ManageSchedule());
-              },
+          if (userType.isAdmin)
+            IconButton(
+              icon: const Icon(Icons.add),
+              onPressed: () => navigator.push(const ManageSchedule()),
             ),
-          ),
         ],
       ),
       body: SingleChildScrollView(
         child: Column(
           children: [
-            // First Header
-            Container(
-              margin: const EdgeInsets.only(top: 16.0),
-              padding: const EdgeInsets.all(16.0),
-              child: const Text(
-                'Active Livestreams',
+            const Padding(
```

Comment:
`Container()` is really nice for combining padding, size constraints, decoration, and alignment into a single constructor.

But if you only need one of those, it's more efficient to use a dedicated widget.

Reply:
Also very good to know, thanks!